swig/python/codegen/codegen.py

```diff
@@ -0,0 +1,1172 @@
+from __future__ import print_function, unicode_literals
+import six
+import sys
+import ast
+import os
+import tokenize
+import warnings
+from six import StringIO
```

Comment:
Use `io.StringIO` instead, which is core python3.

Reply:
Done. Also removed from `setup.py.in`.